recipes/hangar/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "hangar" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85bbb1e8fe0cac56ca86257283a96472bd8410443d1b819d0e6b67f0a0d63214
+
+build:
+  number: 1
+  skip: true  # [py<36]
+  entry_points:
+    - hangar = hangar.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+
+requirements:
+  host:
+    - blosc-hdf5-plugin  # [not win]
+    - click
+    - grpcio
+    - h5py ==2.9.0
+    - msgpack-python ==0.6.1
+    - numpy
+    - pip
+    - protobuf
+    - python
+    - python-blosc
+    - python-lmdb >=0.92
+    - pyyaml
+    - tqdm
+    - wrapt
+  run:
+    - blosc-hdf5-plugin  # [not win]
+    - click
+    - grpcio
+    - h5py ==2.9.0
+    - msgpack-python ==0.6.1
+    - numpy
+    - protobuf
+    - python
+    - python-blosc
+    - python-lmdb >=0.92
+    - pyyaml
+    - tqdm
+    - wrapt
+
+test:
+  imports:
+    - hangar
+    - hangar.diagnostics
+    - hangar.records
+    - hangar.remote
+  commands:
+    - hangar --help
+
+about:
+  home: https://github.com/tensorwerk/hangar-py
+  license: Apache 2.0
```

Really minor but for consistency with the SPDX naming scheme:
```suggestion
  license: Apache-2.0
```

---

Fixed!